Charts: update the 'Custom Annotations' demo description (#1935)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/CustomAnnotations/description.md

> Annotations are containers for images, text blocks, and custom content that display additional information about the visualized data. To create custom annotations, specify the following Chart properties.

Reviewer: > Annotations are containers for images, text blocks, and custom content **that display additional information about** the visualized data.

Unfortunately, it's not quite clear to what this phrase is related exactly. It should be related to "annotations", however, due to a breakup with "images, text blocks, and custom content" it becomes not that obvious.

Therefore, I suggest the following change:

```suggestion
Annotations are designed to show additional information about visualized data. You can display images, text blocks, and custom content in annotations. To create custom annotations, specify the following Chart properties.
```

## annotations[]

> The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses the data source array to create this collection. Each object has an [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) to make the component display annotations relative to specific arguments and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object that stores data from data source.

Reviewer: > This demo uses **the** data source array to create this collection.

Which data source exactly? I'm questioning your choice of article here.

> Each object has an [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) to make the component display annotations relative to specific arguments

I cannot say I understand this. Did you mean that the mentioned argument allows you to position annotations in different ways? If yes, I believe we need to clarify this further.

> a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object that stores data from **data source**.

An article is missing.

Author: I feel like I need to explain how the **argument** property works.
So there are multiple different ways to specify where the annotations should be displayed. You can set coordinates for each annotation, you can display annotations on axes, and you can anchor them to a chart element. Charts have argument axis and value axis. If you want to anchor annotation to a specific chart point, then you need to specify both **value** and **argument** properties (chart coordinates). And if you need to anchor annotation to a series point (series is group of related points, like in this demo there are multiple bars, the whole group of these bars is a series, and each bar is a series point), you need to specify both **argument** and **series** properties, since each series point represents one argument. In this demo, the **series** property is already specified in the **commonAnnotationSettings** object, so it's the same series for all annotations. But each annotation has different argument value (a different series point). So that's why it's necessary to specify the **argument** in the **annotations** array, so the component knows where to display the annotations. If you don't specify the **argument**, the Chart shows annotations on the series, but near the value axis.

I tried not to explain all of it in the demo, since all of this information can be found in the documentation, so I tried to squeeze all of this into one sentence, but I guess I should expand it instead.

Reviewer: Alright, now I understand your point. However, the sentence in question looks as if something is amiss. I'll offer you my version and you'll see if it suits you.

> The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses a data source array to create this collection. Each object has an [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) to make the component display annotations relative to specific arguments and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object that stores data from a data source.

Reviewer: > to make the component display annotations relative to specific arguments

This phrase sounds very vague.

Also, this sentence is quite long and difficult to read and understand per se.

Did I get this right? Does it make sense?

```suggestion
The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses a data source array to create this collection. Each object has the [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object. The [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property positions an annotation at a specific argument value, and the [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object stores data from a data source.
```

Author: I would not say "argument value" since in chart terminology arguments and values are different things.
If I say simply
> The [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property positions an annotation at a specific argument, and the data...

Is it okay?

Reviewer: Yeah, this will do, I guess.

> You can use the [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array to specify configuration for each individual annotation.

Reviewer: An article is missing. However, I do believe that this version would do better:

```suggestion
You can use the [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array to configure each annotation individually.
```

> The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses a data source array to create this collection. Each object has the [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object. The [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property positions an annotation at a specific argument value, and the [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object stores data from a data source.

Author: ```suggestion
The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses a data source array to create this collection. Each object has the [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object. The [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property positions an annotation at a specific argument, and the [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object stores data from a data source.
```

## commonAnnotationSettings

> Use the [commonAnnotationSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/) object to specify settings common for all annotations in the chart. In this object, assign `custom` to the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/#type) property. To anchor annotations to a series point, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/#series) property. Finally, declare the SVG markup in the annotation [template](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#template).

Reviewer: ```suggestion
Use the [commonAnnotationSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/) object to specify settings common to all annotations in the chart. In this object, assign `custom` to the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/#type) property. To anchor annotations to a series point, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/#series) property. Finally, declare the SVG markup in the annotation [template](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#template).
```

> Note that individual [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) settings override the [commonAnnotationSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/). 

Reviewer: Please do not intertwine API member names with ordinary text. This breaks the sentence in terms of grammar. What prevents you from placing a link to the corresponding API members from simple words as anchor text?

I would like to see something like this here:

```suggestion
Note that individual [annotation settings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) override the [common annotation settings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAnnotationSettings/). 
```

## customizeAnnotation

> You can also use the [customizeAnnotation](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#customizeAnnotation) function to customize an individual chart annotation.

Reviewer: You say "use". How should I use it exactly?

Author: I changed the word to 'implement'.

Reviewer: OK
